packages/cli/vite.config.integration.ts

```diff
@@ -12,6 +12,5 @@ export default defineConfig({
     },
     environment: 'node',
     include: ['src/**/*.integration.test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}'],
-    globalSetup: ['global-setup.ts'],
```

Comment:
Is the global setup not needed anymore? It takes care of cleaning `tmp` folder for tests using real file system.

Reply:
No, it is not needed. And I intend to get rid of the `tmp` folder altogether if possible.

The only test using file system is `config-middleware` which only reads configuration files.
`yargs-cli` integration tests should not actively use a file system, only parse parameters.